book_class.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+end
```

Comment:
- [ ] Kindly create a method `add_rental` in the `Book` class.  This method should take `Person`, `Date` and `Book` as parameters. The method should create a new Rental object. This would enable every book object to be able to create a new rental. Remember that `Book` and `Person` have a _**many to many**_ relationship through `Rental`. Therefore both `Book` and `Person` should be able to create a `Rental`.